CONFIGURATION.md

```diff
@@ -116,7 +116,7 @@ Environment Variable | Config File Key | Description | Default
 `SW_APM_PROXY` | `:http_proxy` | Configure an HTTP proxy through which the library connects to the collector. | None
 `SW_APM_SERVICE_KEY` | `:service_key` | API token + service name combination, **required**. |
 `SW_APM_TAG_SQL` | `:tag_sql` | Enable/disable injecting trace context into supported SQL statements. | false
-`SW_APM_TRIGGER_TRACING_MODE` | `:trigger_tracing_mode` | | enable
+`SW_APM_TRIGGER_TRACING_MODE` | `:trigger_tracing_mode` | | `:enable`
```

Comment:
Can you pull in latest from main branch then update based on what was added there please?